peer_mentoring/groups/templates/groups_show_post.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="post-title">
+        <h1>{{post.title}}</h1>
+    </div>
+    <div class="post-author">
+        <h2>{{post.author}}</h2>
+    </div>
+    <div class="post-content">
+        <h4>{{post.content}}</h4>
+    </div>
+
+    {% if comment in comments %}
```

Comment:
You can also use a for...else construct, little nicer:
https://stackoverflow.com/questions/16287695/django-template-language-using-a-for-loop-with-else